homeassistant/helpers/service.py

```diff
@@ -76,3 +94,11 @@ def extract_entity_ids(hass, service_call):
         return group.expand_entity_ids(hass, [service_ent_id])
 
     return [ent_id for ent_id in group.expand_entity_ids(hass, service_ent_id)]
+
+
+def is_valid_service_call(config):
+    """
+    Helper method to validate that a configuration is a valid
+    service call.
+    """
+    return CONF_SERVICE in config or CONF_SERVICE_TEMPLATE in config
```

Comment:
Added on my side already.
 
With this to the validation in a method that returns a boolean makes us loose the information about what went wrong. It could be an idea to return a string instead, and None if all went well. 

Then there is the question about actually validating everything needed for a service call. such as domain_service and service_data. What do you think about creating such a method for a more thorough validation so that we can really trust the configuration after that is done?